whitney/isa.py

```diff
@@ -42,7 +42,12 @@ class ALU(Enum):
     Or = 0x12
     And = 0x13
     XOr = 0x14
-
+    FGetMant     = 0x92
+    FAddIExp     = 0x93
+    FSubExp      = 0x94
+    FCnvExp2F    = 0x95
+    FGetFInt     = 0x96
+    FGetFFrac    = 0x97
```

Comment:
So...we're adding six new ops with opcodes 0x92-0x97? Does this mean the opcode field has expanded from five bits (0x00-0x14) to eight bits (0x00-0x97)? Is there rhyme/reason for the opcode assignments...I thought they were going to be assigned automatically, or is that still TBD? Also I don't see Fadd/Fsub/Fmul, or is that also TBD?

Reply:
Those are already in master... It looks like this PR is based on the version before those were merged in. We're going to need to rebase this on the current master before merging.